src/daisyworld/daisyworld/Patch.java

```diff
@@ -41,7 +48,7 @@ public double calcTemp() {
         double absorbedLuminosity = calcLuminosity();
         double localHeating = absorbedLuminosity > 0 ? Math.log(absorbedLuminosity) * 72 + 80 : 80;
 
-        return (temperature + localHeating) / 2;
+        return (temperature + localHeating) * Params.DIFFUSION_RATE;
```

Comment:
This is to calculate the average value instead of diffusing, should not use diffusion_rate

Reply:
I gonna correct it in the next version.